src/main/java/fi/fmi/avi/archiver/message/MessageValidator.java

```diff
@@ -6,9 +6,8 @@
     /**
      * Validates an aviation message. Throws an exception if validation fails.
      *
-     * @param message
-     *         message
+     * @param message message
      */
-    void validate(AviationMessage message);
+    AviationMessage.Builder validate(AviationMessage.Builder message);
```

Comment:
When modifying a builder, I think there should be no need to return anything. Especially when the return value is not being used. Now that the MessageModifier API is basically the same as this, these could be merged in the future. I'll accept this now.